wp-content/themes/missilethreat/inc/template-tags.php

```diff
@@ -362,7 +362,7 @@ function missiledefense_display_footnotes() {
 			$footnotes = $easyFootnotes->easy_footnote_after_content('');
 
 			if ( $footnotes != '' ) {
-				printf( '<div class="entry-footnotes col-xs-12 collapsible-content-container"><h4 class="post-footer-heading collapsible-title">' . esc_html( 'Footnotes', 'missiledefense') . '</h4><ol class="easy-footnotes-wrapper collapsible-content">%1$s</ol></div>', $footnotes ); // WPCS: XSS OK.
+				printf( '<div class="footnotes col-xs-12 collapsible-content-container"><h2 class="footnotes__heading collapsible-title text--semibold">' . esc_html( 'Footnotes', 'missiledefense') . '</h2><ol class="footnotes__list collapsible-content">%1$s</ol></div>', $footnotes ); // WPCS: XSS OK.
```

Comment:
```suggestion
				printf( '<div class="footnotes collapsible-content-container"><h2 class="footnotes__heading collapsible-title text--semibold">' . esc_html( 'Footnotes', 'missiledefense') . '</h2><ol class="footnotes__list collapsible-content">%1$s</ol></div>', $footnotes ); // WPCS: XSS OK.
```

Remove the `col-xs-12` class since it doesn't mean anything anymore.